src/nvim/ops.c

```diff
@@ -5441,15 +5441,17 @@ static yankreg_T *adjust_clipboard_name(int *name, bool quiet, bool writing)
   if (*name == '*' || *name == '+') {
     if(!eval_has_provider("clipboard")) {
       if (!quiet) {
-        EMSG("clipboard: provider is not available");
+        EMSG("clipboard: Provider not available. See \":h clipboard\" and "
+             "\":CheckHealth\".");
```

Comment:
Could be a bit more terse:

    clipboard: No provider. Try ":CheckHealth" or ":h clipboard".